agent/bench-scripts/pbench-fio

```diff
@@ -322,11 +322,11 @@ function fio_process_options() {
 				# make it absolute
 				client_file=$PWD/$client_file
 			fi
+			local delim=""
 			while read line; do
-				clients="$clients,$line"
+				clients="${clients}${delim}${line}"
+				delim=","
```

Comment:
OK, this code idiom is annoyingly clever.  (I'm not sure whether I want to remember this for future use or forget I ever saw it!)

Reply:
I'd rather keep the code the way it is since it does not need to go back and fix anything up, and it is the same number of lines, and touches the values less.

Reply:
I think I disagree with most of those assertions, but I don't object to this proposed change.

Reply:
(I'm leaving it up to @dbutenhof to resolve this conversation, though. 😉)